feat(chart): remove mirrored openshift image from Longhorn (#9600)
longhorn/longhorn: chart/README.md

| image.longhorn.supportBundleKit.tag | string | `"v0.0.43"` | Tag for the Longhorn Support Bundle Manager image. |
| image.longhorn.ui.repository | string | `"longhornio/longhorn-ui"` | Repository for the Longhorn UI image. |
| image.longhorn.ui.tag | string | `"master-head"` | Tag for the Longhorn UI image. |

> | image.openshift.oauthProxy.repository | string | `""` | Repository for the OAuth Proxy image. Specify the upstream image, for example, quay.io/openshift/origin-oauth-proxy. This setting applies only to OpenShift users. |

Reviewer: ```suggestion
| image.openshift.oauthProxy.repository | string | `""` | Repository for the OAuth Proxy image. Specify the upstream image (for example, "quay.io/openshift/origin-oauth-proxy"). This setting applies only to OpenShift users. |
```

> | image.openshift.oauthProxy.tag | float | `""` | Tag for the OAuth Proxy image. Specify OCP/OKD version 4.1 or later. The Longhorn team has validated version 4.15, available at quay.io/openshift/origin-oauth-proxy:4.15. This setting applies only to OpenShift users. |

Reviewer: ```suggestion
| image.openshift.oauthProxy.tag | float | `""` | Tag for the OAuth Proxy image. Specify OCP/OKD version 4.1 or later (including version 4.15, which is available at quay.io/openshift/origin-oauth-proxy:4.15). This setting applies only to OpenShift users. |
```
